typings.d.ts

```diff
@@ -0,0 +1,43 @@
+interface ITesters {
```

Comment:
Interface the properties?
``
ITester {
 handle:...
 profile:....
}
twitter: ITester
``

?